packages/apps/stock/src/components/templates/supplierArrival/SupplierArrivalTrackingNumberSelect/SupplierArrivalTrackingNumberSelect.js

```diff
@@ -67,48 +57,17 @@ const SupplierArrivalTrackingNumberSelect = ({
     }
   };
 
+  if (!canCreate) {
+    return null;
+  }
```

Comment:
It's functionnally false, canCreate permission should show/hide only the creation part of the component, not the selection part. 
Also why having two components SuppleirArrivalTrackingNumberSelect & SupplierArrivalTrackingNumberSelection which will have the same navigation on creation, permission check,...
try to refactor to have only one component, the only difference should be the onSelectTrackingNumber which can be given in props for both screens that use it.